lib/auth/person.ex

```diff
@@ -225,24 +224,27 @@ defmodule Auth.Person do
       nil ->
         create_person(person)
 
-      ep -> # existing person
+      # existing person
+      ep ->
         merged = Map.merge(AuthPlug.Helpers.strip_struct_metadata(ep), person)
-        {:ok, person} = changeset(%Person{id: ep.id}, merged)
-        # |> IO.inspect(label: "changeset transformed:234")
-        |> Repo.update()
 
-        person # |> IO.inspect(label: "updated person:230")
+        {:ok, person} =
+          changeset(%Person{id: ep.id}, merged)
+          # |> IO.inspect(label: "changeset transformed:234")
```

Comment:
I'm happy for *all* `IO.inspect` to be removed.
Looking forward to adding more advanced Logging https://github.com/dwyl/app/issues/274 👍